MSAL/src/native_auth/network/responses/validator/reset_password/MSALNativeAuthResetPasswordResponseValidator.swift

```diff
@@ -166,8 +173,8 @@ final class MSALNativeAuthResetPasswordResponseValidator: MSALNativeAuthResetPas
 
     private func handleContinueError(_ error: Error, with context: MSIDRequestContext) -> MSALNativeAuthResetPasswordContinueValidatedResponse {
         guard let apiError = error as? MSALNativeAuthResetPasswordContinueResponseError else {
-            MSALLogger.log(level: .error, context: context, format: "resetpassword/continue returned unexpected error type")
-            return .unexpectedError
+            MSALLogger.log(level: .error, context: context, format: "Reset Password Continue: Error was not decoded properly, error: \(error)")
```

Comment:
Whit the previous error format it was clear that "continue" was an endpoint, while with this new format it is isn't. Can you add this information in the log?

Reply:
I have brought it in line with all the other errors of this type, for example on Sign Up we use:
`Sign Up Challenge: Error was not decoded properly`
This one IMO was incorrect

Reply:
What I'm saying is that "sign up challenge" doesn't clarify that "challenge" is an endpoint of the sign up flow. "signup/continue" was clearer. I'm suggesting adding "endpoint" to the log: "Sign Up Challenge endpoint: Error was not decoded properly" or use the / : "signup/challenge: Error was not decoded properly"